src/main/java/com/maybe/maybe/entity/enums/DeskState.java

```diff
@@ -0,0 +1,6 @@
+package com.maybe.maybe.entity.enums;
+
+public enum DeskState {
+    ALL,
```

Comment:
Dont understand status ALL? What it means?
I think there should be: AVAILABLE, NOT_AVAILABLE, ORDERED